charts/templates/coordinator.yaml

```diff
@@ -131,10 +131,10 @@ metadata:
     app.kubernetes.io/component: client-api
     app.kubernetes.io/created-by: {{ .Values.global.createdBy }}
     app.kubernetes.io/name: coordinator-client-api
-    app.kubernetes.io/part-of: marblerun
+    app.kubernetes.io/part-of: {{ .Release.Namespace }}
     app.kubernetes.io/version: {{ .Values.coordinator.version }}
 spec:
-  type: ClusterIP
+  type: NodePort
```

Comment:
Why this change?
There should be no namespace issues with ClusterIP

Reply:
Sorry for that, it is a leftover.  While we were testing it it was easier for us to have that port externally available.